sandy-city-utah/targetsolutions-user-sync
# feat(ts-service): pull users from the targetsolutions api

### What Changed?
  - add user pull functionality to the target solutions service
  - incorporate logging in the application
  - use user pull in main file

### Issues Addressed:
  - P21312-16

### Issue Testing:
  - Manual Developer Testing

### Test Results:
1. Run the process
2. see that target solutions users are pulled in correctly
